core/src/test/scala/com/yahoo/maha/core/RequestModelTest.scala

```diff
@@ -4887,7 +4887,8 @@ class RequestModelTest extends FunSuite with Matchers {
     }
   }
 
-  test("create model should fail when using or filters with fact and dim filter combination") {
+  //No longer valid, we now allow both & split on the backend.
```

Comment:
In that case, do we have a test which tests allowing both?

Reply:
The test is in DruidQueryGeneratorTest & generates both.

Reply:
RequestModel is meant to work across data sources.  If you are changing behavior for creating model, it should be tested here first.  Query generation tests are specific to each engine.